src/main/views/macros/elements/task-list/enum/taskStatus.ts

```diff
@@ -0,0 +1,4 @@
+export enum TaskStatus {
+  Completed = 'Complete',
+  NotStarted = 'Not yet started'
+}
```

Comment:
I am thinking its better if views directory will have only files with *.njk extension. and all *.ts files somewhere else. What do you think?

Reply:
Moved to 'app/case/types' and included tests.